packages/connectivity/ios/Classes/FLTConnectivityPlugin.m

```diff
@@ -156,7 +156,9 @@ - (NSString*)convertCLAuthorizationStatusToString:(CLAuthorizationStatus)status
     case kCLAuthorizationStatusAuthorizedWhenInUse: {
       return @"authorizedWhenInUse";
     }
-    default: { return @"unknown"; }
+    default: {
```

Comment:
This is a change made by running the formatter over the plugins repo (a requirement for submitting a PR).

While it is out of scope of this PR it might pay to fix up the formatting?

Reply:
Actually I don't know what happened here as the formatter is complaining about the change. Reverted

Reply:
maybe your plugin_tool is outdated?

Reply:
I'm using the latest (0.0.32+5)